docs/migration.md

```diff
@@ -44,6 +44,8 @@ consider additional positioning prop support on a case-by-case basis.
 #### @zendeskgarden/react-chrome
 
 - Removed `PRODUCT` type export. Use `IHeaderItemProps['product']` instead.
+- Some subcomponent exports have been deprecated and will be removed in a future major version.
+  Update to subcomponent properties.
```

Comment:
Wondering if we should document the specific mapping here, since it's a bit more nebulous with the number of components in the package.

Reply:
Yeah, I'd recommend copying the mapping from your PR description to a bulleted sublist here.